actionpack/lib/action_controller/metal/request_forgery_protection.rb

```diff
@@ -419,6 +419,12 @@ def protect_against_forgery? # :doc:
       def valid_request_origin? # :doc:
         if forgery_protection_origin_check
           # We accept blank origin headers because some user agents don't send it.
+          raise InvalidAuthenticityToken, "The browser returned a 'null' origin for a request with origin-based" \
+          " forgery protection turned on. This usually means you have the 'no-referrer' Referrer-Policy header " \
```

Comment:
This is as nit-picky as it gets, but it bothers me that this is the only line with a space at the beginning rather than the end. Is it worth just doing a `strip_heredoc` here?